core/contracts/financial-templates/expiring-multiparty/Liquidatable.sol

```diff
@@ -295,6 +295,16 @@ contract Liquidatable is PricelessPositionManager {
             })
         );
 
+        // If this liquidation is a subsequent liquidation on the position, and the liquidation size is larger than minSponsorTokens,
+        // then re-set the liveness. This enables a liquidation against a withdraw request to be "dragged out" if the position is very large.
+        if (
+            positionToLiquidate.withdrawalRequestPassTimestamp > 0 && // this is a subsequent liquidation.
+            positionToLiquidate.withdrawalRequestPassTimestamp < getCurrentTime() && // liveness has not passed yet.
```

Comment:
This should be <= instead of < based on how `withdraw()` handles comparising the request timestamp to `getCurrentTime()`

Reply:
Hmmmm, why would we _only_ do this positions that have not yet passed the liveness? Do we see this being meaningfully helpful to UX? I don't think it necessarily matters that much -- it just seems like it makes the casework more complex and harder to explain, so I think we'd need to have a good reason to _add_ this complexity over just doing it in all withdrawal-based liquidations.

Reply:
My main logic with this was if you had passed the liquidation liveness you should not be able to re-set this again. If this restriction was not added then a passed withdrawal request could have it's liveness re-set. This might actually be desired, in the case that the sponsor is slow to withdraw.

Reply:
> This should be <= instead of < based on how withdraw() handles comparising the request timestamp to getCurrentTime()

Any response to this?

Reply:
> > This should be <= instead of < based on how withdraw() handles comparising the request timestamp to getCurrentTime()
> 
> Any response to this?

Yup. To be strictly consistent I agree with you.